### DISCUSSION TITLE
Czy istnieje możliwość sprawdzenia które z moich ofert mają błędnie wypełniony parametr?

### DISCUSSION THREAD

**Discussion by tomylee0** [allegro/allegro-api#4832] [Suggestions - rejected]
Czy istnieje możliwość sprawdzenia ofert które mają wypełniony obowiązkowy parametr ale jest on wypełniony błędnie? Jeśli nie ma to chciałbym zgłosić sugestię. 

**Generalnie chodzi mi o łatwy sposób wyszukiwania w których ofertach nie uda mi się zaktualizować ilości sztuk albo ceny.** Pomocny jest tutaj zasób /sale/offers/unfilled-parameters ale on podaje tylko parametry puste (przynajmniej problem #4760 uda mi się z pomocą tego zasobu rozwiązać).

**Po co mi to?** Według mojej obserwacji parametry i kategorie w bazie produktowej Allegro zmieniają się. Ale parametry te nie są automatycznie zmieniane w podpiętych ofertach. Oznacza to, że nie mogę zmienić ilości sztuk albo ceny w ofercie z powodu nieaktualnego parametru. Ale dowiem się o tym dopiero w momencie kiedy będę chciał zmienić ilość sztuk lub cenę. A chciałbym o tym wiedzieć wcześniej, żebym mógł zaciągnąć świeże parametry z bazy produktowej i wrzucić je do moich ofert. 

Przy okazji chciałbym zapytać czy zasób /sale/offers/unfilled-parameters zwraca również oferty zakończone? Generalnie też dobrze wiedzieć wcześniej **których ofert nie uda mi się wznowić jeśli nie uaktualnię parametrów**.

Pozdrawiam

**Comment by Lukasz-Zurek** [comment]
@tomylee0 Dziękuję za przesłane propozycje. Zgodnie z naszym [newsem](https://github.com/allegro/allegro-api/issues/4337), przeniosłem Twoją sugestię do zakładki Discussions.

Dodam, że w przypadku [GET /sale/offers/unfilled-parameters](https://developer.allegro.pl/documentation/#operation/getOffersUnfilledParametersUsingGET_1) nie zwracamy ofert zakończonych.

**Comment by PrzemyslawLukanowski** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ nie planujemy sugerowanej zmiany.
